### ISSUE TITLE
alerts contrast

### ISSUE THREAD

**Issue by ajjjjjjjj** [bassjobsen/bootstrap-a11y-theme#1]
alert classes are not affected, and fail at meeting WCAG AAA text contrast ratio 7:1.
